docs/conf.py

```diff
@@ -1,10 +1,12 @@
 import datetime
+import os
 import sys
 
-try:
-    import checkbox_ng
-except ImportError:
-    raise SystemExit("checkbox_ng has to be importable")
+sys.path.append(os.path.abspath(
+    os.path.join(__file__, "../../checkbox-ng")
+))
```

Comment:
Is the `checkbox_ng` module used for anything else other than retrieving the version number? (It might be used by `sphinx.ext.autodoc`, I'm not familiar with this) 

If not, we could just find a way to grab the version number.

Reply:
I think it's required by autosummary and co indeed.